app/components/shared/Table/Table.tsx

```diff
@@ -97,6 +108,10 @@ export const Table = <T extends IdObj>({
     return obj
   }
 
+  paginatedData.map((item) => {
```

Comment:
This map is not needed cause is logging a value and doing nothing more.

Reply:
fixed, ty @carlosvq